addons/l10n_sk/views/report_template.xml

```diff
@@ -1,62 +1,57 @@
-                ID: <span t-field="company.company_registry"/>
-            </li>
-            <li t-if="company.income_tax_id and company.account_fiscal_country_id.code == 'SK'">
-                Tax ID: <span t-field="company.income_tax_id"/>
-            </li>
+            <t t-call="l10n_sk.vat_registry_tax_id_sk_external_layout"/>
         </xpath>
     </template>
 
     <template id="l10n_sk_external_layout_boxed" inherit_id="web.external_layout_boxed">
         <xpath expr="//ul[@name='company_address_list']" position="inside">
-            <li t-if="company.vat and company.account_fiscal_country_id.code == 'SK'">
-                <t t-esc="company.country_id.vat_label or 'Tax ID'"/>:
-                <span t-esc="company.vat"/>
-            </li>
-            <li t-if="company.company_registry and company.account_fiscal_country_id.code == 'SK'">
-                ID: <span t-field="company.company_registry"/>
-            </li>
-            <li t-if="company.income_tax_id and company.account_fiscal_country_id.code == 'SK'">
-                Tax ID: <span t-field="company.income_tax_id"/>
-            </li>
+            <t t-call="l10n_sk.vat_registry_tax_id_sk_external_layout"/>
         </xpath>
     </template>
 
     <template id="l10n_sk_external_layout_striped" inherit_id="web.external_layout_striped">
         <xpath expr="//ul[@name='company_address_list']" position="inside">
-            <li t-if="company.vat and company.account_fiscal_country_id.code == 'SK'">
-                <t t-esc="company.country_id.vat_label or 'Tax ID'"/>:
-                <span t-esc="company.vat"/>
-            </li>
-            <li t-if="company.company_registry and company.account_fiscal_country_id.code == 'SK'">
-                ID: <span t-field="company.company_registry"/>
-            </li>
-            <li t-if="company.income_tax_id and company.account_fiscal_country_id.code == 'SK'">
-                Tax ID: <span t-field="company.income_tax_id"/>
-            </li>
+            <t t-call="l10n_sk.vat_registry_tax_id_sk_external_layout"/>
+        </xpath>
+    </template>
+
+    <template id="l10n_sk_external_layout_bubble" inherit_id="web.external_layout_bubble">
+        <xpath expr="//ul[@name='company_address_list']" position="inside">
+            <t t-call="l10n_sk.vat_registry_tax_id_sk_external_layout"/>
+        </xpath>
+    </template>
+
+    <template id="l10n_sk_external_layout_wave" inherit_id="web.external_layout_wave">
+        <xpath expr="//ul[@name='company_address_list']" position="inside">
+            <t t-call="l10n_sk.vat_registry_tax_id_sk_external_layout"/>
+        </xpath>
+    </template>
+
+    <template id="l10n_sk_external_layout_folder" inherit_id="web.external_layout_folder">
+        <xpath expr="//ul[@name='company_address_list']" position="inside">
+            <t t-call="l10n_sk.vat_registry_tax_id_sk_external_layout"/>
         </xpath>
     </template>
```

Comment:
The extra values should be on the left, if we follow the same approach as in the other l10n
![image](https://github.com/user-attachments/assets/3566ad11-d94e-4b6e-a505-a1a8b77195bc)